apps/web/components/eventtype/CreateEventType.tsx

```diff
@@ -152,7 +152,7 @@ export default function CreateEventTypeButton(props: CreateEventTypeBtnProps) {
         options={props.options}
       />
 
-      <DialogContent className="overflow-y-auto">
+      <DialogContent>
```

Comment:
This didn't work before; but it did break the new scroll behaviour so for this reason I've edited this.